back/engines/free/email_campaigns/spec/fixtures/locales/mailers.fr-FR.yml

```diff
@@ -353,9 +353,9 @@ fr:
       title_added_as_folderadmin: 'Vous êtes à présent gestionnaire de dossiers'
       x_projects: '%{numberOfProjects} projets'
     project_phase_started:
-      cta_view_phase: 'Découvrez cette nouvelle phase'
+      cta_button: 'Découvrez cette nouvelle phase'
```

Comment:
@jamesspeake I think we forgot fr-FR and nl-NL in the fixtures for some campaigns?